docs/stories/hooks/useActiveKeys.stories.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/react';
+import { useActiveKeys } from '@buffetjs/hooks';
+import { Button } from '@buffetjs/core';
+
+import Presentation from '../ui/Presentation';
+import Pre from '../ui/Pre';
+
+const Story = () => {
+  const [enableActiveKeysListener, setEnableActiveKeysListerner] = useState(
+    false
+  );
+  const activeKeys = useActiveKeys(enableActiveKeysListener);
+
+  return (
+    <Presentation
+      title="useActiveKeys"
+      description="Hook to save the user's keyboard events."
+    >
+      <div style={{ marginBottom: 23 }}>
+        <Button
+          primary
```

Comment:
The prop is `color="primary"` or nothing and it will be primary by default (to update in the `Pre` too)